packages/button/Button.stories.tsx

```diff
@@ -61,83 +59,882 @@ export default {
+              color='primary'
+              size='l'
+              shape='oval'
+              disabled={true}
+              withArrow={true}
+              loading={false}
+            >
+              Button text
+            </Button>
+            <Button
+              color='primary'
+              size='l'
+              shape='oval'
+              disabled={true}
+              icon={{ icon: <GradientCircle />, isColored: true }}
+              withArrow={false}
+              loading={false}
+            >
+              Button text
+            </Button>
+            <Button
+              color='primary'
+              size='l'
+              shape='oval'
+              disabled={true}
+              icon={<Tiles />}
+              withArrow={false}
+              loading={false}
+            >
+              Button text
+            </Button>
+          </div>
+        </div>
+
+        {/* Color: secondary */}
+        <div style={colorGroupStyle}>
+          <h5 style={colorTitleStyle}>Color: Secondary</h5>
+          <div style={buttonGroupStyle}>
+            <Button
+              color='secondary'
+              size='l'
+              shape='oval'
+              disabled={false}
+              withArrow={false}
+              loading={false}
+            >
+              Button text
+            </Button>
+            <Button
+              color='secondary'
+              size='l'
+              shape='oval'
+              disabled={false}
+              withArrow={true}
+              loading={false}
+            >
+              Button text
+            </Button>
+            <Button
+              color='secondary'
+              size='l'
+              shape='oval'
+              disabled={false}
+              icon={{ icon: <GradientCircle />, isColored: true }}
+              withArrow={false}
+              loading={false}
+            >
+              Button text
+            </Button>
+            <Button
+              color='secondary'
+              size='l'
+              shape='oval'
+              disabled={false}
+              icon={<Tiles />}
+              withArrow={false}
+              loading={false}
+            >
+              Button text
+            </Button>
+            <Button
+              color='secondary'
+              size='l'
+              shape='oval'
+              disabled={true}
+              withArrow={false}
+              loading={false}
+            >
+              Button text
+            </Button>
+            <Button
+              color='secondary'
+              size='l'
+              shape='oval'
+              disabled={true}
+              withArrow={true}
+              loading={false}
+            >
+              Button text
+            </Button>
+            <Button
+              color='secondary'
+              size='l'
+              shape='oval'
+              disabled={true}
+              icon={{ icon: <GradientCircle />, isColored: true }}
```

Comment:
no need to segregate icons this way, colored ones would ignore color changes, so we can treat all icons as `isColored: false` and spare this prop